### ISSUE TITLE
Echo canceller calibration sometimes very silent

### ISSUE THREAD

**Issue by driesken** [BelledonneCommunications/linphone-android#1712]
**Describe the bug**
Sometimes when running the echo canceller calibration, the generated tones are very silent. When closing Linphone and restarting it, the volume is restored to maximum level.
I suppose this might also affect calls since the remote party sometimes complains of me sounding at a distance.

**To Reproduce**
1. Audio -> Echo cancellation callibration
2. Hear generated tones

**Please complete the following information**
 - Device: Samsung Galaxy S10e
 - OS: Android 12
 - Version of the App 4.7.0
 - Version of the SDK 5.2.0-alpha.211

**SDK logs**
[62b97918c38ee_521e8360f9fcdb76424a.txt](https://github.com/BelledonneCommunications/linphone-android/files/8990664/62b97918c38ee_521e8360f9fcdb76424a.txt)

**Comment by driesken** [comment]
Thanks. I tested the latest master and it appears that my Samsung Galaxy S10e (which is supposed to have AEC) now has an echo delay every time I try to calibrate. Is this as expected?

` MESSAGE Echo calibration estimated delay to be 216 ms`

**Comment by Viish** [comment]
Indeed. I haven't seen it sooner because on my device the hardware echo canceller seems to work all the time, when on Samsung it only works when audio manager's mode is "in communications".
Fix is on the way.

**Comment by driesken** [comment]
Thanks. I've noticed that this is merged into the 5.1 branch but has yet to be approved for the master. I'll try it out afterwards.

**Comment by Viish** [comment]
Yes we're having issues with our CI...

**Comment by driesken** [comment]
I'm not sure if this is fixed yet in latest SDK (5.22.0-alpha.218+65bdf2b). No tones are heard during calibration (same Samsung device).

I've attached some logging.

[no tones during calibration.txt](https://github.com/BelledonneCommunications/linphone-android/files/9234243/no.tones.during.calibration.txt)
